django/contrib/admin/views/main.py

```diff
@@ -174,9 +177,21 @@ def get_filters(self, request):
         may_have_duplicates = False
         has_active_filters = False
 
+        supports_request = func_supports_parameter(
+            self.model_admin.lookup_allowed, "request"
+        )
+        if not supports_request:
+            warnings.warn(
+                f"Add 'request' to the signature of "
+                f"{self.model_admin.lookup_allowed.__self__.__class__.__name__}"
+                f".lookup_allowed(). Support for ModelAdmin subclasses that do not "
+                f"accept this argument is deprecated.",
+                RemovedInDjango60Warning,
+            )
```

Comment:
```suggestion
            warnings.warn(
                f"`request` must be added to the signature of "
                f"{self.model_admin.__class__.__qualname__}.lookup_allowed()",
                RemovedInDjango60Warning,
            )
```

Reply:
This gave a message of:
```
`request` must be added to the signature of ModelAdminTests.test_lookup_allowed_without_request_deprecation.<locals>.ConcertAdmin.lookup_allowed().
```
Which I thought was a bit confusing. So I went with your other suggestion of using `self.model_admin`

Reply:
Thank you, that makes sense :+1: I made an update